src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
good explanation :-) 

Quoting from above:
In other words, the main purpose of each test:
```HtmlHelperTest```: testing the injection and replacement of values
```GodModeTest```: testing the regeneration of HTML file

If the htmlhelper test is to test whether the injection and the replacement of values work properly, shouldn't they be tested individually too? combining them weakens the test, doesn't it? Also, I think we should specify this goal of "testing injection and replacement of values" in comments so that later devs dont remove this test thinking it is a repetition.